en/concept/events_interface.md

```diff
@@ -0,0 +1,133 @@
+# Events Interface
+
+The *Events Interface* provides a system-wide API to inform a user about certain events.
+It is meant to replace existing usage of `mavlink_log_*` calls.
+
```

Comment:
@bkueng Does the MAVLink spec cover the concept of common/managed enums? Specifically, if it is likely that a GCS will configure itself based on events (say display "ready to ARM" when a readiness event is received) then it makes sense that this kind of event is shared between GCS so that nothing needs to be coded twice.
 
@julianoes Perhaps madness, but could/should we reasonably use events as a generic mavlink notifier "in future"? What I mean is that for command protocol we're tdiscussing mechanisms to tell us completion of long running events. We could instead bypass completely and define a Takeoff complete event. I appreciate this depends on events interface being broadly implemented, but is the idea "sane"?

Reply:
This is on the level of events, not mavlink. And yes there is a shared set of enumerations/messages: https://github.com/mavlink/libevents/blob/master/events/common.json

> @julianoes Perhaps madness, but could/should we reasonably use events as a generic mavlink notifier "in future"? What I mean is that for command protocol we're tdiscussing mechanisms to tell us completion of long running events. We could instead bypass completely and define a Takeoff complete event. I appreciate this depends on events interface being broadly implemented, but is the idea "sane"?

I'm not Julian, but my answer: I would not tie it to certain mavlink commands, but something like a 'takeoff complete' event makes sense and is something I intended to use it for. Such events will be defined in the common namespace: https://github.com/mavlink/libevents/blob/master/events/common.json

Reply:
That's great - even though you are not Julian :-).